test/components/views/beacon/RoomLiveShareWarning-test.tsx

```diff
@@ -255,31 +236,29 @@ describe("<RoomLiveShareWarning />", () => {
                     },
                     "$0",
                 );
-                beacon.update(room1Beacon1Update);
+                beacon?.update(room1Beacon1Update);
             });
 
             // update to expiry of new beacon
-            expect(getExpiryText(component)).toEqual("3h left");
+            expect(container).toHaveTextContent("3h left");
         });
 
         it("clears expiry time interval on unmount", () => {
             const clearIntervalSpy = jest.spyOn(global, "clearInterval");
-            const component = getComponent({ roomId: room1Id });
-            expect(getExpiryText(component)).toEqual("1h left");
+            const { container, unmount } = getComponent({ roomId: room1Id });
+            expect(container).toHaveTextContent("1h left");
 
-            act(() => {
-                component.unmount();
-            });
+            unmount();
 
             expect(clearIntervalSpy).toHaveBeenCalled();
         });
 
         it("navigates to beacon tile on click", () => {
             const dispatcherSpy = jest.spyOn(defaultDispatcher, "dispatch");
-            const component = getComponent({ roomId: room1Id });
+            const { container } = getComponent({ roomId: room1Id });
 
             act(() => {
-                component.simulate("click");
+                fireEvent.click(container.firstChild! as Node);
```

Comment:
Do you think it might be better to use userEvent here? I think it's a bit more representative of real world use as it creates more events (for a given interaction, like `userEvent.click` than firing a single event like this. 

(Won't be copying this comment for the other instances of fireEvent)

Reply:
Thought this would be a simple search and replace, but all tests are failing weirdly enough